tests/ui/sanitizer/large-stack-size-issue-83060.rs

```diff
@@ -0,0 +1,21 @@
+// This test checks that allocating a stack size of 1GB or more results in a warning
+//@build-pass
```

Comment:
Not sure if we should turn the warning and test with build-fail ( since CI core dumps) or the warning needs to be an error. I believe it's the latter, but want to be sure. @estebank